examples/spark/pom.xml

```diff
@@ -0,0 +1,151 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Can we migrate this to Gradle so we still have just one build system for all the examples

Reply:
Definitely understand. I'm not at all experienced with gradle, but I'll give it a shot later today.